bazel/external/ubuntu_packages/packages.bzl

```diff
@@ -38,7 +40,7 @@ def download_ubuntu_packages():
     )
     http_file(
         name = "libbyteman-java",
-        urls = ["https://storage.googleapis.com/pl-public-test/ubuntu-debs/libbyteman-java_4.0.18-1_all.deb"],
+        urls = ["https://storage.googleapis.com/pl-public-test/ubuntu-debs/libbyteman-java_%s-1_all.deb" % libbyteman_version],
```

Comment:
With the upgrade to ubuntu 22.04, we now download packages from the ubuntu repositories and then store them in our own gcs mirror. Unfortunately, it seems like ubuntu 22.04 only has version 4.0.18 of this library. If you can use the 4.0.18 version that would be much better since its guaranteed by ubuntu to be compatible with the other libs we install. If there's a reason you can't upgrade let me know and we can work something out

Reply:
My testing on the PR description proves that byteman is working as expected, so that isn't a concern 👍